SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -568,12 +568,20 @@ class DecorativeMeshFileRep : public DecorativeGeometryRep {
 // no default constructor
 explicit DecorativeMeshFileRep(const std::string& meshFileName) 
 :   meshFile(meshFileName) {
+    pmesh = nullptr;
 }
 
 const std::string& getMeshFile() const {
     return  meshFile;
 }
 
+const PolygonalMesh& getMesh() {
```

Comment:
Also do you think this method should be const? Its semantics are const but it has a mutable write-once cache. It might be cleaner to use mutable for them. OTOH this works and the top level is const where it matters.